### PR TITLE
Feat(profile)/transfer

### PR DESCRIPTION
**Pull request by bal7hazar**

### CODE REVIEW

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]
Thanks for refining this flow. Code looks good to me. Just need to test it but not sure how to solve the iframe issue in the preview on this PR. Unless we find a quick solution, maybe we should merge and test in main branch preview with Sepolia account right after. We need to disable the sending button in case when games uses old version of controller without `sendBackward/foward` method tho.

File: packages/profile/src/components/inventory/token/token.tsx [hunk 1]
```diff
@@ -30,10 +30,16 @@ import { formatEther } from "viem";
...
 import { TokenPair } from "@cartridge/utils/api/cartridge";
+import { usePostHog } from "posthog-js/react";
+import { useEffect } from "react";
 
 export function Token() {
   const { address } = useParams<{ address: string }>();
   const location = useLocation();
+  const posthog = usePostHog();
+  useEffect(() => {
+    posthog.setPersonPropertiesForFlags({ address });
```
Comment (tarrencev) [inline packages/profile/src/components/inventory/token/token.tsx:41]: can we move this to the connection where we set other posthog metadata?

Comment (JunichiSugiura) [reply packages/profile/src/components/inventory/token/token.tsx]: moved it to `useAccount` hook

**Review by broody — APPROVED** [review]
Erc20 transf erworks locally for me, I think should be fine to release and address comments

**Review by JunichiSugiura — APPROVED** [review]
LGTM

**Pull request merged**